roles/kubernetes/node/tasks/main.yml

```diff
@@ -16,6 +16,7 @@
 - import_tasks: install.yml
   tags:
     - kubelet
+    - kube-proxy
```

Comment:
Why this tag?

Reply:
I wanted to use the same behaviour than kubelet but as kube-proxy is ran from a container I guess it's not needed. I'll remove it

Reply:
fixed in 953e2abcd5426183d9ceef3facb8fe957f2a6d82